### PR TITLE
Add PNG development output

### PR DESCRIPTION
**Pull request by jelly**
This PR adds a new config option which if set to `display_class = display_png`  is useful for local development where a `display.png` is written on every draw. This is useful for local development without an e-ink monitor.

### CODE REVIEW

File: config.ini.example [hunk 1]
```diff
@@ -6,6 +6,7 @@ quiet = no
 debug = yes
 
 ## Display specific
+display_class = display
```
Comment (Kartoffel) [inline config.ini.example:9]: This option is named `display`, see [`run,py`, line 45](https://github.com/Kartoffel/infodisplay/pull/3/commits/962e020cf0729cd311cbfb2087d729856990292b#diff-d6af0459a37d985953d7040c14f53feb3b9cc9e58b543aa3c2b80256d276c5e0R45).

File: display_png.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+logger = logging.getLogger(__name__)
+
+
+class Display:
+    def __init__(self, _cfg):
+        self.logger = logging.getLogger(__name__)
+        # FIXME: height from config file?
+        self.filename = 'display.png'
+        self.height = 600
+        self.width = 800
```
Comment (Kartoffel) [inline display_png.py]: Yes please, let's move width and height to config parameters.
The default display class gets the width and height of the display automatically, so the config parameters should be optional. Width 800 and height 600 seems like sensible defaults.

**Review by Kartoffel — CHANGES_REQUESTED** [review]
Super useful feature, thanks!
I've added some feedback to your commit, please inspect :)

File: config.ini.example (hunk 1 above)
Comment (jelly) [reply config.ini.example:9]: I'll make it `display_class` as `display` is a bit ambigious.

**Review by Kartoffel — APPROVED** [review]
Looks good, thanks!

**Pull request merged**